docs/Account/Account-template.markdown

```diff
@@ -0,0 +1,84 @@
+ * Login. After your user has registered and verified their registration, this operation allows them to log in to your app.
+
+##The Registration Step
+To register new users, use the `ORKRegistrationStep` class. The resulting view shows a title and helpful registration text. There, the user can type an email address and password, along with other optional information.
+
+The following example specifies a registration step that includes requesting the user's full name, gender, and birthday:
+
+	    ORKRegistrationStep *registrationStep = [[ORKRegistrationStep alloc] initWithIdentifier:@"identifier"
+                                                                                      title:@"Account Registration" text:@"Please register at this point."
+                                                                                    options:ORKRegistrationStepIncludeGivenName |                                             ORKRegistrationStepIncludeFamilyName | ORKRegistrationStepIncludeGender |ORKRegistrationStepIncludeDOB];
+    ORKOrderedTask *task = [[ORKOrderedTask alloc] initWithIdentifier:@"registrationTask" steps:@[registrationStep]];
+    ORKTaskViewController *taskViewController = [[ORKTaskViewController alloc] initWithTask:task taskRunUUID:nil];
+    taskViewController.delegate = self;
+    [self presentViewController:taskViewController animated:YES completion:^{}];
+
+
+Here's how the registration view appears:
+
+![](Registration.png)
+
+After all of the information has been properly set, the user can touch the Done button to proceed. Your app is responsible for sending the user-provided data to a server to perform the actual registration.
+
+##The Verification Step
+Email verification is a common way to validate proper registration for a user. After the user registers, your backend server can send an email to the provided email address.
+
+To enable email verification in your app, use a combination of the `ORKVerificationStep` class along with a subclass of the `ORKVerificationStepViewController`. Subclassing `ORKVerificationStepViewController` allows you to provide custom behavior when the user taps the Resend Email Verification button. The following code demonstrates how to set up your subclass:
+
+	@interface MyVerificationStepViewController : ORKVerificationStepViewController
+	@end
+
+	@implmentation MyVerificationStepViewController
+
+	- (void)resendEmailButtonTapped;
+	{
+   		// perform custom logic here 
+	}
+
+	@end
+
+Next, create a verification step and pass in the subclass object:
+
+    ORKVerificationStep *verificationStep = [[ORKVerificationStep alloc] initWithIdentifier:@"identifier"
+    text:@"Please verify."
+    verificationViewControllerClass:[MyVerificationStepViewController class]];
+
+You can then insert the step into your `ORKOrderedTask` object. When the step is executed, it instantiates the view controller and shows the following:
+
+![](Verification.png)
```

Comment:
I think it'd be nicer if these images wouldn't include the *iOS Simulator* window title bar. That's achieved by capturing them using `Command + S` from within the simulator, instead of using the `Command + Shift + 4, Space` command.